components/navigation/navigation.css

```diff
@@ -0,0 +1,31 @@
+.navbar {
+  width: 100%;
+  margin: 0 auto;
+  padding: 1rem;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  background-color: var(--primary-color-light);
+  box-shadow: 0px 0px 1rem 4px var(--bs-light);
+}
+.navbar img {
+  width: 4.5rem;
+  display: block;
+  margin-right: 1.5rem;
+}
+.search-bar {
+  display: flex;
+  width: 25rem;
+  padding: 0.75rem;
+  border-radius: 4px;
+  background-color: var(--secondary-font-color);
+}
+.search-input {
```

Comment:
You can remove the outline from the input

Reply:
Actually removing the outline (i.e. using `outline: none`) is bad for accessibility.

Reply:
probably you can make outline:0.

Reply:
Done